website/blog/2020-03-19-cobalt.md

```diff
@@ -0,0 +1,162 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.2 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are happy to announce the release of Metals v0.8.2, which main purpose is
+adding support for new Scala version 2.12.11. Additionally, we included a couple
+of recent fixes.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">55</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">14</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">1</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/20?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/) and
+give Metals a try!
+
+- support for Scala 2.12.11
+- recent minor improvements
+
+## Miscalenous improvements
+
+- handle fatal exceptions when evaluating worksheets
+- fix issue with false positive warning about Bloop version change
+- fix rename for hierarchy with generics
+- ignore return type when renaming override methods and fields
+- fix issue when docstrings where not update after changes in the workspace
+- it's now possible to navigate to Scala sources even outside a build target or
+  in a worksheet
+
```

Comment:
- fix issue with string alignment when multiple multiline strings were present in a file